theme/theme.inc

```diff
@@ -154,7 +157,7 @@ function template_process_islandora_solr_metadata_description(array &$variables)
               $field['description_data']['truncation']['ellipsis'],
               $field['description_data']['truncation']['word_safe'],
               $field['description_data']['truncation']['min_wordsafe_length'],
-              " "
```

Comment:
Replacing a " " with a newline is a regression. I'd be fine with hard coding the space for the time being seeing as it only applies in a very specific edge case where the description is being truncated and I don't think warrants having another variable for just this specific option.